src/rules/no-array-length-in-deps.ts

```
@@ -0,0 +1,276 @@
+						const indent = lastNewlineIndex >= 0
+							? prefixBeforeProgram.slice(lastNewlineIndex + 1)
+							: prefixBeforeProgram;
+
+						let importText = '';
+						const needUseMemo = !isUseMemoImported(sourceCode);
+						const needStableHash = !isStableHashImported(sourceCode);
+						if (needUseMemo) importText += `${indent}import { useMemo } from 'react';\n`;
+						if (needStableHash)
+							importText += `${indent}import { stableHash } from 'functions/src/util/hash/stableHash';\n`;
+
+						if (importDecls.length === 0) {
```

ESLint

[error] Replace `↹↹↹↹↹↹` with `············`

(prettier/prettier)